### PR TITLE
feat: updated `configure()` API to enable/disable tooltip campaigns feature (SDKCF-6076)

### PR DESCRIPTION
**Pull request by maureenorea-clores**
# Checklist
- [x] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [x] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data **before every commit**, including API endpoints and keys
- [ ] I checked that all possible internal and platform/library exceptions are gracefully handled and **will not** crash the host app
- [ ] I checked for open & known issues (especially crashes) for any newly added platform APIs or libraries
- [ ] I tested non-trivial changes on a real device
- [x] I added ticket/changes in changelog
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

File: inappmessaging/USERGUIDE.md [hunk 1]
```diff
@@ -397,6 +397,16 @@ override fun onRequestPermissionsResult(requestCode: Int, permissions: Array<Str
 
 **<font color="red">Note:</font>** Please make sure that `trackPushPrimer()` API is only called for devices with Android 13 or higher OS since push notification permission request is not available for devices running in lower OS versions.
 
+### <a name="tooltip-campaigns"></a> #6 Tooltip Campaigns
```
Comment (vincentdelmundo) [inline inappmessaging/USERGUIDE.md:400]: nit: add a note or link to this section in the SDK Integration section.

File: inappmessaging/USERGUIDE.md [hunk 2]
```diff
@@ -397,6 +397,16 @@ override fun onRequestPermissionsResult(requestCode: Int, permissions: Array<Str
...
+
+Tooltip feature is currently in beta testing; its features and behaviour might change in the future.
+Please refer to the internal guide for more information.
+
+To enable tooltips you must set `enableTooltipFeature` flag to true when calling `configure()`.
+```kotlin
+InAppMessaging.instance().configure(enableTooltipFeature = true)
```
Comment (vincentdelmundo) [inline inappmessaging/USERGUIDE.md:407]: This require `context` right?

File: inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/data/repositories/CampaignRepository.kt [hunk 3]
```diff
@@ -47,28 +48,33 @@ internal abstract class CampaignRepository {
...
-                }
-
+            for (newCampaign in messageList.filterMessages(ignoreTooltips)) {
                 val updatedCampaign = updateCampaign(newCampaign, oldList)
                 messages[updatedCampaign.getCampaignId()] = updatedCampaign
             }
             saveDataToCache()
         }
 
+        private fun List<Message>.filterMessages(ignoreTooltips: Boolean): List<Message> {
```
Comment (vincentdelmundo) [inline inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/data/repositories/CampaignRepository.kt:72]: nice!!

**Review by vincentdelmundo — CHANGES_REQUESTED** [review]
Looks good. Please see minor comments.

File: inappmessaging/USERGUIDE.md (hunk 1 above)
Comment (maureenorea-clores) [reply inappmessaging/USERGUIDE.md:400]: Update

File: inappmessaging/USERGUIDE.md (hunk 2 above)
Comment (maureenorea-clores) [reply inappmessaging/USERGUIDE.md]: Thanks, I missed that

**Review by vincentdelmundo — APPROVED** [review]

**Pull request merged**